### ISSUE TITLE
Learning Path Table of Contents Left-Sidebar does not wrap text

### ISSUE THREAD

**Issue by joemoore** [vmware-tanzu/tanzu-dev-portal#1289]
**Describe the bug**
The article Table of Contents along the side of learning paths does not wrap, causing the sidebar links to run into the main content.

**To Reproduce**
1. Go to 'https://tanzu.vmware.com/developer/outcomes/application-development/'
1. Observe the left-hand TOC 

**Screenshots**
![Screen Shot 2021-09-24 at 1 16 39 PM](https://user-images.githubusercontent.com/13995/134728894-159dd967-aa5c-44b4-a102-9ff5cea283c2.png)

**Expected behavior**
Pretty wrapping

**Comment by jonasmit** [comment]
I noticed this as well.  The easiest fix is to change:
sidebar-outcomes.html:
Line 6
```
<li class="td-sidebar-nav__section-title text-nowrap">
```
to
```
<li class="td-sidebar-nav__section-title">
```

But @BrianMMcClain - Do you want to have some sort of way of making the wrapping more clear? We don't want it to look like an additional nav element

![Screen Shot 2021-09-30 at 1 28 46 PM](https://user-images.githubusercontent.com/551536/135503156-319e212a-1102-4f9d-b41a-4f5dad31e938.png)

**Comment by jonasmit** [comment]
#1306

**Comment by BrianMMcClain** [comment]
Hey @joemoore @jonasmit, this is actually being addressed in the nav-reorg PR #1224 that's being merged tomorrow, so if it works for you two I'm going to link that PR to this issue and close your related PR just to ease the merging. Appreciate this!
